core/sqf/monitor/linux/monitor.cxx

```diff
@@ -1089,6 +1090,22 @@ int main (int argc, char *argv[])
         tmpptr++;
     }
 
+    // Remove the domain portion of the name if any
```

Comment:
This is the third copy I've seen of code to remove the domain portion. Maybe they should be refactored into a separate function?